grpc/grpc.cpp

```diff
@@ -0,0 +1,40 @@
+#include "grpc.h"
+
+#include "library/elliptics.h"
+#include "server.hpp"
+#include "thread_pool.hpp"
+
+struct dnet_grpc_io {
+    std::unique_ptr<ioremap::grpc::server_t> server;
+    std::unique_ptr<ioremap::grpc::thread_pool_t> thread_pool;
+    std::unique_ptr<::grpc::CompletionQueue> completion_queue;
+};
+
+int dnet_grpc_io_start(struct dnet_node *node, const char *address, unsigned thread_num) {
+    using namespace ioremap::grpc;
+
+    try {
+        node->grpc = new dnet_grpc_io;
+
+        if (address) {
+            auto error = server_t::make(node->grpc->server, node->grpc->completion_queue, address, *node);
+            if (error) {
+                return error;
+            }
+        } else {
+            node->grpc->completion_queue = std::make_unique<grpc::CompletionQueue>();
```

Comment:
Why? Some strange partial initialization, that doesn't lead to any workability. Either grpc is properly configured and then properly iniaialized, or, fully unitinialized if bad config.

Reply:
`grpc->completion_queue` will be used for clients too.
Ok. I'll add separate function for initialization of `dnet_grpc_io` as client. It'll be more understandable.

Reply:
If it's some forward undertaking, that isn't properly developed yet, let's put TODO's with short description (why, what) - and no questions would occur.

Reply:
c88a982